packages/core/src/eventUtils.ts

```diff
@@ -10,7 +10,7 @@ import { DoneActorEvent, DoneStateEvent, ErrorActorEvent } from './types.ts';
  */
 export function createAfterEvent(delayRef: number | string, id?: string) {
   const idSuffix = id ? `#${id}` : '';
-  return { type: `xstate.after(${delayRef})${idSuffix}` } as const;
+  return { type: `xstate.after.${delayRef}.${idSuffix}` } as const;
```

Comment:
`id` is always provided from what I can see, let's make that argument required and drop this conditional expression (just inline the `id` directly in the `type`)